app/models/appointment.rb

```diff
@@ -22,6 +22,11 @@ class Appointment < ApplicationRecord
     other: 'other'
   }
 
+  enum appointment_type: {
+    manual: 'manual',
+    automatic: 'automatic'
+  }
```

Comment:
```suggestion
  }, _prefiix: true
```

Reply:
Added `_prefix: true` to the `appointment_type` enum.